test/monitoring/aws-kinesis/KinesisFirehoseMonitoring.test.ts

```diff
@@ -32,6 +32,21 @@ test("snapshot test: all alarms", () => {
         maxRecordsThrottledThreshold: 5,
       },
     },
+    addIncomingBytesExceedThresholdAlarm: {
+      Critical: {
+        safetyThresholdLimit: 0.8,
+      },
+    },
+    addIncomingRecordsExceedThresholdAlarm: {
+      Critical: {
+        safetyThresholdLimit: 0.8,
+      },
+    },
+    addIncomingPutRequestsExceedThresholdAlarm: {
+      Critical: {
+        safetyThresholdLimit: 0.8,
```

Comment:
Nitpick: would be nice to see some varying numbers to capture a difference between them in the snapshot.

Reply:
done